cscareerhub/megabot
# [feature] Restructure code, basic set up of DB, and client startup

## Purpose of pull request

Restructure code, update dependencies, basic set up of MongoDB/mongoose, and set up client startup

## Pull request checklist


## Testing instructions

Set up a `.env` with a bot token and mongoDB uri and run `yarn start`. Things should build, run, and log correctly. The tester bot you're using should also start up in its server.
